spiralfilm/core.py

```
@@ -115,6 +124,7 @@
         Returns:
             The result of the API call.
         """
+        self.check_token_limit(messages)
```

Ruff (0.8.2)

`messages` may be undefined, or defined from star imports

(F405)